images/metering-ansible-operator/roles/meteringconfig/tasks/configure_reporting.yml

```diff
@@ -18,3 +18,22 @@
   vars:
     kube_minor_version: "{{ (kube_version.stdout | from_json | json_query('serverVersion.minor')).rstrip('+') }}"
     kube_version_at_least_1_14: "{{ ((kube_minor_version | int) >= 14) | bool }}"
+
+---
```

Comment:
This is going to be problematic as the `---` directive signals the start of a YAML file [1], and there's already that same directive at the start of the `configure_reporting.yml` file.

[1] https://camel.readthedocs.io/en/latest/yamlref.html